appveyor.yml

```diff
@@ -1,20 +1,27 @@
+version: '{build}'
+
+environment:
+  matrix:
+    - JAVA_HOME: C:\Program Files\Java\jdk1.8.0
+
 install:
-  - cinst -y jdk8
-  - cinst -y ant -ignoreDependencies
-  - set JAVA_HOME=C:\Program Files\Java\jdk1.8.0
-  - set PATH=%JAVA_HOME%\bin;%PATH%
-  - set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
-  - python --version
-  - java -version
-  - ant -version
+  - cmd: choco install ant -y -f -i
+  - cmd: refreshenv
+  - cmd: set BUCK_EXTRA_JAVA_ARGS=-Djna.nosys=true
+  - cmd: set GOROOT=
+  - cmd: set PATH=%PATH:C:\go\bin;=%
+  - cmd: set PATH=%PATH:C:\go\bin=%
```

Comment:
How does this work? I assume it's for removing it from the path, but I can't find any reference for the syntax.